### PR TITLE
Change run commands to take a display and delay value and set them in wbi

### PR DESCRIPTION
**Pull request by tnederlof**
Closes https://github.com/sol-eng/wbi/issues/115

### CODE REVIEW

File: internal/system/command.go [hunk 1]
```diff
@@ -27,15 +30,19 @@ func RunCommand(command string) error {
 }
```
Comment (samcofer) [inline internal/system/command.go:30]: We need to print the command when the command fails. You can test it by running wbi as non-root. The failure message gives you an error, but doesn't communicate how it checked if you're root.

File: internal/system/command.go [hunk 2]
```diff
@@ -27,15 +30,19 @@ func RunCommand(command string) error {
...
 
-	fmt.Println("Running command: " + command)
-	// sleep for 3 seconds to allow the user to read the command
-	time.Sleep(3 * time.Second)
 	cmd := exec.Command("/bin/sh", "-c", command)
+
 	out, err := cmd.CombinedOutput()
 	if err != nil {
-		return "", fmt.Errorf("issue in CombinedOutput: %w", err)
+		return "", fmt.Errorf("issue running command: %w", err)
```
Comment (samcofer) [inline internal/system/command.go:45]: We need to print the command when the command fails. You can test it by running wbi as non-root. The failure message gives you an error, but doesn't communicate how it checked if you're root.

**Review by samcofer — COMMENTED** [review]
Overall looks good, but we need to print the actual command that failed, so that we can troubleshoot by running the command outside the program.

File: internal/system/command.go (hunk 1 above)
Comment (tnederlof) [reply internal/system/command.go:30]: Ah good call, I will take a look at this. Not just the error but include the command that failed. Should the command that failed be printed (stdout) or included in the error?

Comment (samcofer) [reply internal/system/command.go:30]: I'd include it in the error output and make sure it's clearly visible as the actual command that was run.

Comment (tnederlof) [reply internal/system/command.go:30]: I can include the command in the error but keep in mind error is a high bar it means something went wrong with the run command itself, not that it returned any stderr.

File: internal/system/command.go (hunk 2 above)
Comment (tnederlof) [reply internal/system/command.go:45]: When running wbi as non-root it doesnt actually trigger a proper error it just looks at strings.TrimSpace(idOutput) != "0" and prints of so and error. We could change that one for instance but I wanted to highlight that the error in `RunCommand` and `RunCommandAndCaptureOutput` only trigger when something went wrong executing the `cmd.Run()` or `cmd.CombinedOutput()` commands.

File: internal/system/command.go (hunk 1 above)
Comment (samcofer) [reply internal/system/command.go:31]: I think we should return the command in any error case. I wasn't specific enough. I guess my unspecific take is, if we are kicking the user out of the flow, then we should display the error. id -u being the example case.

Comment (tnederlof) [reply internal/system/command.go:31]: The more I thought on this one I am less sure printing the command is always a good idea. For most user for example in the root case they just need to know they aren't root, some people might be confused by seeing `id -u`. I guess the middle ground there is give a nice error and say by the way this was the command we used?

**Comment by tnederlof** [comment]
@samcofer took another stab at this. Are these the type of errors you were looking for?

**Review by samcofer — APPROVED** [review]
Good to merge!

**Pull request merged**